backend/canisters/local_group_index/c2c_client/src/lib.rs

```diff
@@ -2,6 +2,7 @@ use canister_client_macros::*;
 use local_group_index_canister::*;
 
 // Updates
+generate_c2c_call!(c2c_can_push_notifications);
```

Comment:
Nit - the comment is "Updates" but this is a query

Reply:
Fixed